airflow/www/views.py

```diff
@@ -4102,6 +4103,12 @@ def action_set_running(self, drs: List[DagRun]):
         """Set state to running."""
         return self._set_dag_runs_to_active_state(drs, State.RUNNING)
 
+    @action('set_skipped', "Set state to 'skipped'", '', single=False)
+    @action_has_dag_edit_access
+    def action_set_skipped(self, drs: List[DagRun]):
+        """Set state to skipped."""
+        return self._set_dag_runs_to_active_state(drs, State.SKIPPED)
```

Comment:
DAGRuns has only 3 states: queued, running and failed. If I understand correctly, you are setting dagrun state to skipped here

Reply:
I Incorrectly implemented for DAGRuns instead of TaskInstance. Fixed this and tested it locally. Works as intended on the taskinstance/list/ page.